packages/dapp/src/components/Navbar/SubMenuComponent.tsx

```diff
@@ -20,145 +22,145 @@ import {
 
 interface NavbarSubMenuProps {
   open: boolean;
-  isOpenSubMenu: boolean;
-  setOpenSubMenu: Dispatch<SetStateAction<string>>;
-  isSubMenu: boolean;
   label: string;
   suffixIcon?: JSX.Element | string;
   prefixIcon?: JSX.Element | string;
   checkIcon?: boolean;
   url?: string;
+  prevMenu: MenuKeys;
   subMenuList: MenuListItem[];
-  triggerSubMenu: string;
+  triggerSubMenu: MenuKeys;
 }
 
 const SubMenuComponent = ({
   open,
-  isOpenSubMenu,
-  setOpenSubMenu,
-  isSubMenu,
+  prevMenu,
   label,
   triggerSubMenu,
   subMenuList,
 }: NavbarSubMenuProps) => {
   const theme = useTheme();
   const isDarkMode = theme.palette.mode === 'dark';
-  const menu = useMenu();
+  const [openNavbarSubMenu, onOpenNavbarSubMenu] = useMenuStore(
+    (state) => [state.openNavbarSubMenu, state.onOpenNavbarSubMenu],
+    shallow,
+  );
+
+  function handleBackSpace(event: KeyboardEvent<HTMLDivElement>) {
+    if (event.key === 'Backspace') {
```

Comment:
Do we want to include `Esc` key here?

Reply:
@chybisov I actually tried it and it was not working out of the box because the Escape let´s the user quit the menu. I believe this also makes sense - so I will leave it with Backspace, but if you have some good argument on why to include the Escape button, I will try to make it work.